pull_request_template.md

```diff
@@ -9,9 +9,9 @@ Fixes #(issue)
 Please delete options that are not relevant.
 
 - [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
+- [x] New feature (non-breaking change which adds functionality)
```

Comment:
Please revert the changes here, the template is automatically used when you open a new PR - no need to update it here

Reply:
Will do!